01-city-tours/src/app.js

```diff
@@ -0,0 +1,40 @@
+import React, {useState} from 'react'
+import Navbar from './components/header'
+import Footer from './components/footer'
+import Card from './components/card'
+import cards from "./components/cards"
+
+
+function App () {
+
+const[cardsList, setList] = useState(cards)
+
+function deleteCard (id) {
+              const filteredCards = cardsList.filter((card)=>card.id !==id)
```

Comment:
Hey @liuba-larousse, is it on purpose having 8 spaces as indentation? Usually the normal is 2 or 4 (personally I prefer 2 nowadays. In the past I used 4).

You can customize that by clicking in the bottom footer on vscode "spaces 8" and change it to 2 or 4 spaces.
![image](https://user-images.githubusercontent.com/14869087/89728695-248cac00-da27-11ea-9eea-44ea94de8385.png)

Also, it seems like you don't have "auto format" enabled on save. To do it go to settings and check "format to save". You may need to restart vscode.

![image](https://user-images.githubusercontent.com/14869087/89728726-5a319500-da27-11ea-9dfe-8e123f99ac7f.png)